addons/website_sale/views/templates.xml

```diff
@@ -1565,7 +1565,8 @@
                 <td colspan="3" class="text-center text-xl-right noborder">
                 <span class=''>
                     <t t-set='force_coupon' t-value="website_sale_order.pricelist_id.code or request.params.get('code_not_available')"/>
-                    <t t-if="not force_coupon">
+                    <t t-set='sale_coupon_active' t-value="request.env['ir.module.module'].sudo().search([('name', '=', 'sale_coupon'), ('state', '=', 'installed')])"/>
+                    <t t-if="not force_coupon and sale_coupon_active">
```

Comment:
- Please reread your commit messages before sending to review...
- ... and no it is not ok to do a search to check an external module is installed ... that part should be moved to sale_coupon but if it crashes in stable, I'd say the correct fix is to check that code is in the fields.

Reply:
Yes I got kind of confused in this...

Reply:
The code is part of website sale but is used in coupon only...

Reply:
@rdeodoo

Reply:
discussed with fja, pricelist can act as a coupon code when the pricelist has an ecommerce promotional code